feature/platform/transceiver/tests/zr_tunable_parameters_test/zr_tunable_parameters_test.go

```diff
@@ -286,3 +347,15 @@ func opticalChannelFromPort(t *testing.T, dut *ondatra.DUTDevice, p *ondatra.Por
 	tr := gnmi.Get(t, dut, gnmi.OC().Interface(p.Name()).Transceiver().State())
 	return gnmi.Get(t, dut, gnmi.OC().Component(tr).Transceiver().Channel(0).AssociatedOpticalChannel().State())
 }
+
+// vendorOpticalChannelOperMode returns the Vendor-specific mode identifier a given vendor.
+func vendorOpticalChannelOperMode(t *testing.T) uint16 {
```

Comment:
Can you please instead make this a flag? Test should be vendor agnostic and Operational-mode value for the test must be passed as a flag

Reply:
Could we instead use switch dut.Vendor() ? , e.g. https://github.com/openconfig/featureprofiles/blob/main/feature/system/gnmi/cliorigin/tests/cli_origin_test/cli_origin_test.go#L97

Reply:
Can you please keep it a flag for this test? Will prefer we don't have to update and publish every single mode for each vendor every time a new mode is enabled or qualified.

Are you seeing any downsides to making this a flag?

Reply:
sure, we can set as flag.
What should be the default value in the flag for this, can it be 0 ?

Reply:
default should be 1.